src/classes/RD2_Constants.cls

```diff
@@ -53,6 +53,15 @@ public inherited sharing class RD2_Constants {
         Disable_All_Installments
     }
 
+    /**
+    * @description Enum to specify asynchronous behavior for the creation of the first installment opportunity
+    */
+    public enum FirstInstallmentAsyncOptions {
```

Comment:
`FirstInstallmentOppCreateOptions` (or `InstallmentOppCreateFirstOptions` ) since async is just one of the options.